4/Dockerfile

```diff
@@ -46,6 +58,8 @@ RUN yum install -y centos-release-scl-rh && \
     rpm -V $INSTALL_PKGS && \
     yum clean all -y
 
+EXPOSE 8080
```

Comment:
Why moving `EXPOSE` instruction?

Reply:
Based on the discussion with @TomasTomecek , I will move it back.